syntax = "proto3";

package hello;

service FraudDetectionService {
    rpc ValidateOrder (FraudDetectionRequest) returns (FraudDetectionResponse);
}

message FraudDetectionRequest {
    string name = 1;
}

message FraudDetectionResponse {
    bool isOk = 1;
}
